### DISCUSSION TITLE
IOS not showing controls

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3423] [Q&A]
Hi all,
I am trying to make a test for a client with a type of Variant of changing the textures on the fly.
My problem is that on android in working as expected BUT on IOS is not even showing the controls
I do not know if it is a model viewer problem or an HTML/CSS.

You can try my project here!
[https://new-window-dinno.glitch.me](https://new-window-dinno.glitch.me)

Any help is much appreicieted

**Comment by elalish** [comment]
I assume you're talking about in AR? This should work consistently in 3D mode on the website. Please ask Apple (@grorg) to implement WebXR in WebKit so this is possible in AR too. Currently your only option for AR on iOS is Quick Look which is outside of the browser and so don't support any of this kind of customization.

**Comment by adrianposteuca** [comment]
Thanks, but can I use this ( https://modelviewer.dev/examples/scenegraph/#createTexturesExample)
" iOS Quick Look reflects these texture changes so long as the USDZ is auto-generated." Can I use this to reflect the changes in textures in IOS?

Fo it reflects the new texture in AR mode?

**Reply by elalish** [reply]
Yes, when you launch Quick Look without an `ios-src` we will send it the current glTF configuration. You just won't be able to change it once you're in Quick Look.

**Comment by adrianposteuca** [comment]
Perfect ,  the only way is to change in the viewer and launch it in AR without ios-scr.

This is a good work around to use a configurator for iOS!
Thanks

Adrian Posteucă

CEO Stakeborg NFT (Blockchain Technology)
CEO ConceptFactory( Immersive Technology)
Bucharest/ London

±40 730 586 103
________________________________
